cmd/homi/docker/service/validator.go

```diff
@@ -159,6 +161,11 @@ var validatorTemplate = `{{ .Name }}:
 {{- end }}
         echo 'ADDITIONAL="$$ADDITIONAL --debug --metrics --prometheus"' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
         /klaytn-docker-pkg/bin/k{{ .NodeType }}d start
+{{- if .UnlockGenesisAccount }}
+        sleep 1
+        ken attach --exec "personal.importRawKey('{{ .NodeKey }}', '')" http://localhost:{{ .RPCPort }}
+        ken attach --exec "personal.unlockAccount('{{ .Address }}', '')" http://localhost:{{ .RPCPort }}
```

Comment:
If there is no time parameter, it will unlock for 300 * time.Second, so I recommend setting it to a large value.
```suggestion
        ken attach --exec "personal.unlockAccount('{{ .Address }}', '', 99999999)" http://localhost:{{ .RPCPort }}
```